src/pages/projects.js

```diff
@@ -378,11 +378,12 @@ const ProjectDescriptions = () => (
         </div>
         <div className="project-description">
           <p>
-            Major:
+            Major:{" "}
             <a href="https://github.com/cilium/ebpf">
               <b>ebpf</b>
-            </a>{" "}
-            Emerging:
+            </a>
+            <br>
```

Comment:
```suggestion
            <br />
```
^ This should fix the complaints from the Netlify build.